intern/remote/boss/remote.py

```diff
@@ -683,6 +683,21 @@ def list_coordinate_frames(self):
         cf = CoordinateFrameResource(name='')
         return self._list_resource(cf)
 
+    def get_channel(self, chan_name, coll_name, exp_name):
```

Comment:
I thought about this, but thought it'd be better to be consistent with the order used to instantiate resources.